Project.toml

```diff
@@ -67,7 +67,7 @@ RuntimeGeneratedFunctions = "0.5.12"
 SciMLBase = "2.57.2"
 Setfield = "1"
 # StructuralIdentifiability = "0.5.8"
-SymbolicUtils = "2.1.2, 3.3.0"
+SymbolicUtils = "< 3.8"
```

Comment:
This one is incorrect? It doesn't do anything about the fact that it was a patch to NaNMath that was already released?

Reply:
This was merged before any fix was released or made (and while the fix was being put in originally as a SymbolicUtils PR).

I'll remove it in https://github.com/SciML/Catalyst.jl/pull/1168 if tests are now passing.